src/csv/CHERI_CSR.csv

```diff
@@ -30,5 +30,6 @@ direct write if address didn't change","✔","","","","Zcmt","Jump Vector Table
 "stdc","0x163","","S","SRW, <<asr_perm>>","tag=0, otherwise undefined","","","","","","","{cheri_default_ext_name}, S-mode","Supervisor Trap Data Capability (scratch register)","","","","","","","","","","","","","","","","","","","","",""
 "ddc","0x416","","U","URW","<<infinite-cap>>","","","","✔","","","{cheri_default_ext_name}","User Default Data Capability","","","","","","","","","","","","","","","","","","","","",""
 "dinfc","0x7bd","","D","DRW","<<infinite-cap>>","","","","","","✔","Sdext","Source of <<infinite-cap>> capability in debug mode, writes are ignored","","","","","","","","","","","","","","","","","","","","",""
-"utidc","0xca0","utid","U","URO","tag=0, otherwise undefined","N/A","N/A","","","","✔","Zstid","Read-only copy of stidc","","","","","","","","","","","","","","","","","","","","",""
-"stidc","0x580","stid","S","SRW, <<asr_perm>>","tag=0, otherwise undefined","Update the CSR using <<SCADDR>>.","direct write","","","","✔","Zstid","Secure thread ID","","","","","","","","","","","","","","","","","","","","",""
+"utidc","0x480","utid","U","Read: U, Write: U, <<asr_perm>>","tag=0, otherwise undefined","Update the CSR using <<SCADDR>>.","direct write","","","","✔","Zstid","User thread ID","","","","","","","","","","","","","","","","","","","","",""
```

Comment:
I think this is the only case of a U-mode register needing ASR? Should we change the description of ASR as now we say it's for _privileged_ register access?

Reply:
I think it is the only CSR that needs ASR permission. This is how ASR is described now: https://github.com/riscv/riscv-cheri/blob/9661d31152451f0aa406ad74ecbc4959cb5d06e5/src/cap-description.adoc?plain=1#L121-L122

Did you mean this one or something else? @tariqkurd-repo

Reply:
yes - that one - util[c] isn't _privileged_ so we'd need to update the definition - assuming that _privileged_ means M-mode / S-mode accessible.

Reply:
For what it's worth, I think we were always planning to exempt `mcycle`, `minstret` etc from ASR checks as well. I don't know to what extend we need that, or whether they're redundant with the `cycle`, `instret` "unprivileged" registers. @jrtc27 ?

Reply:
maybe it's time to update the definition then?

Reply:
@francislaus can we change

`Access System Registers Permission (ASR):: Allow access to privileged CSRs.`

to 

`Access System Registers Permission (ASR):: Allow access to privileged CSRs and <<ustid>>, <<ustidc>>.`

and I think we need another issue to review ASR more generally.
but at least this makes the ASR statement correct.

Reply:
The following description matches my understanding of this PR:

```
Access System Registers Permission (ASR):: Allow read and write access to all privileged (M-mode and S-mode) CSRs with the following exceptions:
. <<utid>>, <<utidc>>, <<stid>>, <<stidc>>, <<mtid>>, <<mtidc>> all require ASR access for writing and not for reading
``